src/Datagrid/SimplePager.php

```diff
@@ -109,6 +109,19 @@ public function getResults($hydrationMode = null)
         return $this->results;
     }
 
+    /**
+     * NEXT_MAJOR: Remove this method.
+     */
+    public function getResults($hydrationMode = null)
```

Comment:
so this `$hydrationMode` argument will not work anymore, right?

Are we ok with this BC break considering that its not part of the `PagerInterface` contract anyway?

Reply:
I'll remove the BC break